App/index.js

```diff
@@ -0,0 +1,146 @@
+const verde = document.getElementById('verde');
+const rojo = document.getElementById('rojo');
+const amarillo = document.getElementById('amarillo');
+const azul = document.getElementById('azul');
+
+const btnIniciar = document.getElementById('botonIniciar');
+const NIVEL_INICIAL = 1;
+const ULTIMO_NIVEL = 10;
+const TIEMPO_PASAR_DE_NIVEL = 2000;
+const TIEMPO_INICIAR_SIGUIENTE_NIVEL = 700;
+const TIEMPO_ILUMINACION_SECUENCIA = 1000;
+const TIEMPO_ILUMINAR_COLOR = 450;
+class Juego {
+  constructor() {
+    this.iniciarJuego = this.iniciarJuego.bind(this);
```

Comment:
Necesitamos hacer bind de este método? Solo pregunto ;)
Te dejo un articulo para que mires https://stackoverflow.com/questions/41391288/why-is-javascript-bind-necessary

Reply:
Si, ya que al quitarlo el juego no sigue el orden de los metodos de las clases